--- /etc/tinyproxy/tinyproxy.conf
+++ /etc/tinyproxy/tinyproxy.conf
@@ -221,7 +221,7 @@
 # The order of the controls are important. All incoming connections are
 # tested against the controls based on order.
 #
-Allow 127.0.0.1
+#Allow 127.0.0.1

 # BasicAuth: HTTP "Basic Authentication" for accessing the proxy.
 # If there are any entries specified, access is only granted for authenticated
@@ -306,8 +306,8 @@
 #
 # The following two ports are used by SSL.
 #
-#ConnectPort 443
-#ConnectPort 563
+ConnectPort 443
+ConnectPort 80

 #
 # Configure one or more ReversePath directives to enable reverse proxy