--- a/entrypoint-initializer.sh
+++ b/entrypoint-initializer.sh
@@ -60,4 +60,12 @@ EOD
   python3 manage.py loaddata initial_surveys
   python3 manage.py installwatson
   exec python3 manage.py buildwatson
+else
+cat <<EOD | python manage.py shell
+import os
+from django.contrib.auth.models import User
+u = User.objects.get(username=os.getenv('DD_ADMIN_USER'))
+u.set_password(os.getenv('DD_ADMIN_PASSWORD'))
+u.save()
+EOD
 fi
