--- entrypoint-initializer.sh
+++ entrypoint-initializer.sh
@@ -123,4 +123,12 @@
   # checks the deduplication config at startup
   echo "Startup Deduplication Configuration"
   python3 manage.py validatededupeconfig
+else
+cat <<EOD | python manage.py shell
+import os
+from django.contrib.auth.models import User
+u = User.objects.get(username=os.getenv('DD_ADMIN_USER'))
+u.set_password(os.getenv('DD_ADMIN_PASSWORD'))
+u.save()
+EOD
 fi