diff --git a/nginx/nginx.conf b/nginx/nginx.conf
index 9f0345af..b3094e81 100644
--- a/nginx/nginx.conf
+++ b/nginx/nginx.conf
@@ -17,6 +17,12 @@ http {
   upstream uwsgi_server {
     include /run/uwsgi_server;
   }
+  # X-Forwarded-For is empty iff coming from GKE healthchecks;
+  # we then set Host header to "healthcheck", to be allowed by Django
+  map $http_x_forwarded_for $forwarded_host {
+    default   $host;
+    ""        healthcheck;
+  }
   server {
     listen 8080;
     location = /50x.html {
@@ -28,6 +34,7 @@ http {
     location / {
       include /run/uwsgi_pass;
       include /etc/nginx/wsgi_params;
+      uwsgi_param HTTP_HOST $forwarded_host;
       uwsgi_read_timeout 1800;
     }
     error_page 500 502 503 504 /50x.html;
